src/Bindings/PluginManager.cpp

```diff
@@ -1733,22 +1734,35 @@ AStringVector cPluginManager::GetFoldersToLoad(cSettingsRepositoryInterface & a_
 		InsertDefaultPlugins(a_Settings);
 	}
 
-	// Get the list of plugins to load:
 	AStringVector res;
-	auto Values = a_Settings.GetValues("Plugins");
-	for (auto NameValue : Values)
+
+	// Get the old format plugin list, and migrate it:
+	auto OldValues = a_Settings.GetValues("Plugins");
+	for (auto NameValue : OldValues)
 	{
 		AString ValueName = NameValue.first;
 		if (ValueName.compare("Plugin") == 0)
 		{
 			AString PluginFile = NameValue.second;
 			if (!PluginFile.empty())
 			{
-				res.push_back(PluginFile);
+				a_Settings.DeleteValue("Plugins", ValueName);
+				a_Settings.SetValue("Plugins", PluginFile, "1");
```

Comment:
If I have a plugin called `Plugin` then I might write `Plugin=1`. This would then replace it with `1=1`.

Reply:
This is why code reviews are great.